packages/office-ui-fabric-react/src/components/ChoiceGroup/examples/ChoiceGroup.Custom.Example.Styles.ts

```diff
@@ -0,0 +1,19 @@
+import { IStyle } from '@uifabric/styling';
+
+interface IChoiceGroupCustomExampleStyles {
+  root?: IStyle;
+  dropdown: IStyle;
+}
+
+const choiceGroupCustomExampleStyle: IChoiceGroupCustomExampleStyles = {
```

Comment:
Example styles should go in the same file as the example (otherwise export still won't work), and to simplify things we typically use just `mergeStyleSets` rather than the `getStyles/classNamesFunction` approach. In this case:

```
const classNames = mergeStyleSets({
  root: {
    display: 'flex',
    alignItems: 'baseline'
  },
  dropdown: {
    marginBottom: '0px',
    marginLeft: '5px'
  }
});
```